front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -52,40 +50,84 @@ export default function ManageTrainSchedule({ setDisplayTrainScheduleManagement
+        <span className="ml-2">{t('tabs.rollingStock')}</span>
+      </>
+    ),
+    withWarning: rollingStockID === undefined,
+    label: t('tabs.rollingStock'),
+    content: <RollingStockSelector />,
+  };
+
+  const tabPathFinding = {
+    title: (
+      <>
+        <img src={pahtFindingPic} alt="path finding" height={24} />
+        <span className="ml-2">{t('tabs.pathFinding')}</span>
+      </>
+    ),
+    withWarning: pathFindingID === undefined,
+    label: t('tabs.pathFinding'),
+    content: (
+      <div className="row no-gutters">
+        <div className="col-xl-6 pr-xl-2">
+          {mustUpdatePathfinding !== undefined && <Itinerary mustUpdate={mustUpdatePathfinding} />}
+        </div>
+        <div className="col-xl-6">
+          <div className="osrd-config-item mb-2">
+            <div className="osrd-config-item-container osrd-config-item-container-map">
+              <Map />
+            </div>
+          </div>
+        </div>
+      </div>
+    ),
+  };
+
+  const tabSimulationSettings = {
+    title: (
+      <>
+        <img src={simulationSettings} alt="simulation settings" height={24} />
+        <span className="ml-2">{t('tabs.simulationSettings')}</span>
+      </>
+    ),
+    label: t('tabs.simulationSettings'),
+    content: (
+      <div className="row no-gutters">
+        <div className="col-lg-6 pr-lg-2">
+          <ElectricalProfiles />
+          <SpeedLimitByTagSelector />
+          <PowerRestrictionSelector />
+        </div>
+      </div>
+    ),
+  };
+
+  /* // WIP to be added in next PR
+  const tabAllowances = {
+    title: (
+      <>
+        <img src={allowancesPic} alt="allowances" height={24} />
+        <span className="ml-2">{t('tabs.allowances')}</span>
+      </>
+    ),
+    label: t('tabs.allowances'),
+    content: 'marges',
+  }; */
 
-  function confirmButton() {
-    return trainScheduleIDsToModify ? (
-      <button
-        className="btn btn-warning"
-        type="button"
-        onClick={() =>
-          submitConfUpdateTrainSchedules(
-            dispatch,
-            t,
-            setIsWorking,
-            trainScheduleIDsToModify,
-            setDisplayTrainScheduleManagement
-          )
-        }
-      >
-        <span className="mr-2">
-          <FaPen />
-        </span>
-        {t('updateTrainSchedule')}
-      </button>
-    ) : (
-      <button
-        className="btn btn-primary"
-        type="button"
-        onClick={() => submitConfAddTrainSchedules(dispatch, t, setIsWorking)}
-      >
-        <span className="mr-2">
-          <FaPlus />
-        </span>
-        {t('addTrainSchedule')}
-      </button>
-    );
-  }
+  const tabAddingSettings = {
+    title: (
+      <>
+        <img src={addingSettingsPic} alt="adding settings" height={24} />
+        <span className="ml-2">{t('tabs.addingSettings')}</span>
+      </>
+    ),
+    label: t('tabs.addingSettings'),
+    content: !trainScheduleIDsToModify && <TrainAddingSettings />,
```

Comment:
```suggestion
    content: isEmpty(trainScheduleIDsToModify) && <TrainAddingSettings />,
```

Reply:
trainScheduleIDsToModify could be undefined, so isEmpty returns an error, impossible to do :(